tests/e2e/performance/test_fio_benchmark.py

```diff
@@ -141,98 +197,119 @@ def codespeed_push(self):
 
 
 @performance
-@pytest.mark.parametrize(
-    argnames=["interface", "io_pattern"],
-    argvalues=[
-        pytest.param(
-            *[constants.CEPHBLOCKPOOL, "sequential"],
-            marks=pytest.mark.polarion_id("OCS-844"),
-        ),
-        pytest.param(
-            *[constants.CEPHFILESYSTEM, "sequential"],
-            marks=pytest.mark.polarion_id("OCS-845"),
-        ),
-        pytest.param(
-            *[constants.CEPHBLOCKPOOL, "random"],
-            marks=pytest.mark.polarion_id("OCS-846"),
-        ),
-        pytest.param(
-            *[constants.CEPHFILESYSTEM, "random"],
-            marks=pytest.mark.polarion_id("OCS-847"),
-        ),
-    ],
-)
 class TestFIOBenchmark(E2ETest):
     """
     Run FIO perf test using ripsaw benchmark
 
     """
 
-    def test_fio_workload_simple(self, ripsaw, es, interface, io_pattern):
+    def ripsaw_deploy(self, ripsaw):
         """
-        This is a basic fio perf test
+        Deploy the benchmark operator (formally ripsaw) CRD
 
-        """
+        Args:
+            ripsaw (obj): benchmark operator object
 
-        # Deployment ripsaw
-        log.info("Deploying ripsaw operator")
+        """
+        log.info("Deploying benchmark operator (ripsaw)")
         ripsaw.apply_crd("resources/crds/" "ripsaw_v1alpha1_ripsaw_crd.yaml")
-        if interface == "CephBlockPool":
-            sc = constants.CEPHBLOCKPOOL_SC
-        else:
-            sc = constants.CEPHFILESYSTEM_SC
 
-        # Create fio benchmark
-        log.info("Create resource file for fio workload")
-        fio_cr = templating.load_yaml(constants.FIO_CR_YAML)
+    def es_info_backup(self, es):
+        """
+        Saving the Original elastic-search IP and PORT - if defined in yaml
 
-        # Saving the Original elastic-search IP and PORT - if defined in yaml
-        if "elasticsearch" in fio_cr["spec"]:
-            backup_es = fio_cr["spec"]["elasticsearch"]
+        Args:
+            es (obj): elasticsearch object
+
+        """
+
+        # for development mode use the Dev ES server
+        if dev_mode:
+            if "elasticsearch" in self.fio_cr["spec"]:
+                self.fio_cr["spec"]["elasticsearch"] = {
+                    "server": "10.0.144.152",
+                    "port": 9200,
```

Comment:
please save port and server address in defaults.py

Reply:
done